### ISSUE TITLE
Unable to view an uploaded file in AML 

### ISSUE THREAD

**Issue by HollyJoyPhillips** [reapit/foundations#964]
**Summary**: After uploading an image/file to a any section in AML, I am unable to view the image after saving. It is not saving the image but instead when clicked the file links opens a new page of AML.

**Location:** https://dev.aml-app.reapit.cloud/ 

**Steps to replicate:** 
1.	Upload a save an image file against any record in AML (Primary ID for example).
2.	Try and view the image you have uploaded and observe the link will take you to a new tab such as https://dev.aml-app.reapit.cloud/RPT20000020

**Expected Behaviour:** When clicking on a saved image, it should open said image in a new tab to view 

**See video:** https://www.dropbox.com/s/iro482eg759ea3t/AMLImage.mp4?dl=0

**Comment by tanphamhaiduong** [comment]
@willmcvay , @HollyJoyPhillips  this ticket has been blocked because I'm clarifying the way to get document image url.

**Comment by tanphamhaiduong** [comment]
Ticket blocked by #1144

**Comment by plittlewood-rpt** [comment]
1144 is now resolved. Accept header is required and should be set to application/octet-stream. The Swagger documentation should be updated shortly

**Comment by HollyJoyPhillips** [comment]
Hi @trankhacvy, I uploaded a file (PNG) to the primary ID of this record (https://dev.aml-app.reapit.cloud/checklist-detail/RPT17000039), saved and closed the modal. 

Opened the modal and clicked on the file. When it finished downloading, I got the following message when trying to open it: 

![image](https://user-images.githubusercontent.com/52408607/82542634-3eb29e80-9b4a-11ea-94eb-8e21e3c24c1e.png)


The file downloaded with no file extension. 

![image](https://user-images.githubusercontent.com/52408607/82542653-45d9ac80-9b4a-11ea-908b-f5e969c32a62.png)
